src/Symfony/Bundle/TwigBundle/Controller/ExceptionController.php

```diff
@@ -63,7 +61,7 @@ public function showAction(Request $request, FlattenException $exception, DebugL
         $code = $exception->getStatusCode();
 
         return new Response($this->twig->render(
-            (string) $this->findTemplate($request, $request->getRequestFormat(), $code, $showException),
+            $this->findTemplate($request, $request->getRequestFormat(), $code, $showException),
```

Comment:
this is a protected method, meaning it is an extension point. I think we should keep the string casting for now, for cases where a child class returns a ``TemplateReferenceInterface`` (which is likely as it was the expected output).

Casting the ``TemplateReferenceInterface`` will give you a template name, and so will work.